GDAPI/GDAPI/Enumerations/IDMigrationMode.cs

```diff
@@ -1,6 +1,9 @@
-﻿namespace GDAPI.Enumerations
+﻿using System;
+
+namespace GDAPI.Enumerations
 {
     /// <summary>Represents the mode of an ID migration operation.</summary>
+    [Obsolete("Use the GDAPI.Enumerations.GeometryDash.LevelObjectIDType enum instead.")]
```

Comment:
just saying `Use the LevelObjectIDType enum instead.` would be fine imo

Reply:
It's meant to be descriptive as to where exactly the new enum is located. It's also just an obsoletion message meant to be temporary until the obsoleted enum is completely discarded from everywhere.